### DISCUSSION TITLE
Reporting showing entries for other customers although user is not in the team

### DISCUSSION THREAD

**Discussion by Vipale** [kimai/kimai#3239] [General]
**Describe the bug**
Reporting shows working hours of another users (intended) of a customer (not intended) in which current user is not part of the team.

A customer wanted to have an overview of the hours I did for them. (so they know how many hours I already did for them in the current month, before giving me new assignments)
Easiest way seemed to give them limited access to kimai.
As I could not find a better way, I set up a new user for them, added them as a teamlead for themselves (the cutsomer) and created a new role which only has Timesheet (other) -> view_other_timesheet as permission. So they could see my working hours for them.
This works fine, except that I dont really need
- Dashboard
- My times
- Weerkly working hours
- Calendar
for this account.

In Timesheets they can only see my working hours for themselves as expected, 
But in Reporting they can see all my working hours for all my customers.

If there is a better/easier way to achieve all of this, please let me know. 

**Additional context**
 - Kimai version: 1.19.1
 - PHP version: 8.0.17

**Comment by kevinpapst** [comment]
You are trying to use Kimai for a use-case that was not meant to be.

I don't think this is a bug, because teamleads need to understand what their members are working on, no matter for which customer/project. But as a matter of fact you have at least two options as workaround. 

1. There is a plugin for what you want: https://www.kimai.org/store/derpilzcc-readonly-access-bundle.html

2. Give permissions for the project detail report instead of the working time report.

**Reply by Vipale** [reply]
> You are trying to use Kimai for a use-case that was not meant to be.
> 
> I don't think this is a bug, because teamleads need to understand what their members are working on, no matter for which customer/project. But as a matter of fact you have at least two options as workaround.

Totally agree on using Kimai in a way it is not meant to be used.
I still think a teamlead shouldn't necessarily be able to see what I am working on. This is something that would be nice to be configurable. Timesheets already do that, they only show entries for customers were the user is team member of. I guess that's what felt inconsistent.

>     1. There is a plugin for what you want: https://www.kimai.org/store/derpilzcc-readonly-access-bundle.html

This looks very promising, I will check it out.

>     2. Give permissions for the project detail report instead of the working time report.

I will have a look into that as well. 

Thanks
